### PR TITLE
RuleTest example

### PR DESCRIPTION
**Pull request by jacknagz**
Add an example using RuleTest to the starter-kit so overrides can be properly tested

### CODE REVIEW

File: overrides/aws_guardduty.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-from pypanther import LogType
+from pypanther import LogType, RuleTest
+from pypanther.rules.aws_guardduty_rules.aws_guardduty_high_sev_findings import AWSGuardDutyHighSeverityFinding
 from pypanther.wrap import exclude, include
 
 sensitive_aws_ervices = {"s3", "dynamodb", "iam", "secretsmanager", "ec2"}
```
Comment (rohantandon) [inline overrides/aws_guardduty.py:5]: Spelling on this variable needs to be corrected

File: overrides/aws_guardduty.py [hunk 2]
```diff
@@ -15,8 +16,21 @@ def guard_duty_discovery_filter(event):
     return event.get("type").startswith("Discovery")
 
 
+guard_duty_high_sev_test = RuleTest(
+    name="UnauthorizedAccess, High-Sev Finding, dnyamoDB",
```
Comment (rohantandon) [inline overrides/aws_guardduty.py:20]: Formatting + spelling cleanup here

**Comment by rohantandon** [comment]
The `poetry.lock` file appears to have had a bad update - could you try regenerating it?

**Review by maxrichie5 — APPROVED** [review]
Good to go once lock file is fixed

**Pull request merged**